### DISCUSSION TITLE
Ideas

### DISCUSSION THREAD

**Discussion by LADSoft** [LADSoft/OrangeC#779] [Ideas]
This is a good place to share ideas for things you would like to see, or talk about problems you are having that you would like to see fixed.

**Comment by ghost** [comment]
I think `omake` could be better than this: https://github.com/rmyorston/pdpmake (it doesn't even support make jobs). What about improving `omake` so it could correctly handle make jobs and write a better documentation for it?

**Comment by LADSoft** [comment]
The version of omake about to be released with the compiler improves upon the existing 'jobs' functionality in several ways and fixes bugs as well; over all it is more like GNU make than previous versions were, while using a little less windows-specific code under the hood.    As far as documentation I didn't want to rewrite the GNU MAKE manual lol...   and it didn't seem like a good idea to just include the gnu make documentation as-is as there may be features that are missing as well as it has been adapted for windows (e.g. path separators are handled a little different).

**Comment by ghost** [comment]
Too few people use your compiler so there are a lot of bugs not yet discovered. If you could have this done you could expose yourself to a lot more users: https://github.com/LADSoft/OrangeC/issues/603 https://github.com/LADSoft/OrangeC/issues/700
